{"uuid": "812aee68-4e0f-4ffa-ab4c-16f2cbdffe6e", "children": ["9930abee-b745-404f-97fa-a072388e2973"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741860497285, "stop": 1741860497285}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 113, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741860497558, "stop": 1741860497565}], "start": 1741860497285, "stop": 1741860497565}